[2025-03-11T09:07:38.862+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-11T09:07:38.933+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tiki_extract_dag.crawl_the-thao-da-ngoai manual__2025-03-11T08:53:49.647616+00:00 [queued]>
[2025-03-11T09:07:38.959+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tiki_extract_dag.crawl_the-thao-da-ngoai manual__2025-03-11T08:53:49.647616+00:00 [queued]>
[2025-03-11T09:07:38.961+0000] {taskinstance.py:2867} INFO - Starting attempt 2 of 4
[2025-03-11T09:07:39.017+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): crawl_the-thao-da-ngoai> on 2025-03-11 08:53:49.647616+00:00
[2025-03-11T09:07:39.052+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=265) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-11T09:07:39.059+0000] {standard_task_runner.py:72} INFO - Started process 282 to run task
[2025-03-11T09:07:39.065+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'tiki_extract_dag', 'crawl_the-thao-da-ngoai', 'manual__2025-03-11T08:53:49.647616+00:00', '--job-id', '540', '--raw', '--subdir', 'DAGS_FOLDER/tiki_extract_dag.py', '--cfg-path', '/tmp/tmpdhn5oi3l']
[2025-03-11T09:07:39.069+0000] {standard_task_runner.py:105} INFO - Job 540: Subtask crawl_the-thao-da-ngoai
[2025-03-11T09:07:39.263+0000] {task_command.py:467} INFO - Running <TaskInstance: tiki_extract_dag.crawl_the-thao-da-ngoai manual__2025-03-11T08:53:49.647616+00:00 [running]> on host 5c8190d7da62
[2025-03-11T09:08:50.156+0000] {job.py:229} INFO - Heartbeat recovered after 71.60 seconds
[2025-03-11T09:08:50.337+0000] {local_task_job_runner.py:266} INFO - Task exited with return code -9. For more information, see https://***.apache.org/docs/apache-***/stable/troubleshooting.html#LocalTaskJob-killed
[2025-03-11T09:08:50.601+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-11T09:08:50.606+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
